index.js

```diff
@@ -40,7 +40,7 @@ module.exports = function markdown(config) {
     }
 
     var md = new Remarkable(options);
-    var ctx = merge({}, options, this.context, context);
+    var ctx = merge({}, options, this, this.context, context);
```

Comment:
@jonschlinkert Tests are still passing, even if there could be a specific one for Assemble.